addons/l10n_ec/models/account_chart_template.py

```diff
@@ -9,11 +9,14 @@ class AccountChartTemplate(models.Model):
 
     def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
         res = super(AccountChartTemplate, self)._prepare_all_journals(acc_template_ref, company, journals_dict=journals_dict)
-        for r in res:
-            if r.get('code') == 'INV':
-                r.update({'name': '001-001 ' + r.get('name'),
-                          'l10n_ec_entity': '001',
-                          'l10n_ec_emission': '001',
-                          'l10n_ec_emission_address_id': company.partner_id.id,
-                          })
+        for journal in res:
+            # TODO figure out why the "account.chart.template" from l10n_ec_edi is being called
+            # TODO move this code to l10n_ec_edi's "account.chart.template" ?
```

Comment:
@odoostan I don't know if you were asking me, if so: We setup some journals in l10n_ec and others in l10n_ec_edi because some fields are not available in l10n_ec, so we decided to setup only the sale journal for which all fields are already available.

Keep also in mind that in l10n_ec_edi we couldn't call _prepare_all_journals() and had to use generate_journals(), because _prepare_all_journals() is called only before installing the chart of accounts, but at the time of installation of  l10n_ec_edi the chart of accounts was already installed, so a little trick was needed. It happens in other localizations too.

I suggest not no move this code because all fields needed are already available in this module.